math/minuit2/inc/Minuit2/FCNGradAdapter.h

```diff
@@ -56,7 +56,15 @@ class FCNGradAdapter : public FCNGradientBase {
    }
    // forward interface
    // virtual double operator()(int npar, double* params,int iflag = 4) const;
-   bool CheckGradient() const { return false; }
+   bool CheckGradient() const override { return false; }
+
+   GradientParameterSpace gradParameterSpace() const override {
```

Comment:
As mentioned above, I would add in this class the flag to distinguish external from internal gradient parameter space